lighthouse-core/config/default-config.js

```diff
@@ -516,6 +517,7 @@ const defaultConfig = {
         {id: 'is-on-https', weight: 2, group: 'pwa-installable'},
         {id: 'service-worker', weight: 1, group: 'pwa-installable'},
         {id: 'installable-manifest', weight: 2, group: 'pwa-installable'},
+        {id: 'ios-pwa-icon', weight: 1, group: 'pwa-installable'},
```

Comment:
Weight?

Reply:
1 seems about right, it's definitely not harder than adding a service worker :)